File: TOKI/Course_Pemrograman Dasar/05 Percabangan/E.py
n = int(input())

if n >= 10000:
    print("puluhribuan")
elif n >= 1000:
    print("ribuan")
elif n >= 100:
    print("ratusan")
elif n >= 10:
    print("puluhan")
elif n >= 1:
    print("satuan")

File: TOKI/Course_Pemrograman Dasar/05 Percabangan/E.cpp
#include <cstdio>

int main() {
    int angka;
    scanf("%d", &angka);
    if (angka < 10) {
        printf("satuan\n");
    } else if (angka < 100) {
        printf("puluhan\n");
    } else if (angka < 1000) {
        printf("ratusan\n");
    } else if (angka < 10000) {
        printf("ribuan\n");
    } else if (angka < 100000) {
        printf("puluhribuan\n");
    }
    return 0;
}